jmclient/setup.py

```diff
@@ -9,7 +9,7 @@
       author_email='',
       license='GPL',
       packages=['jmclient'],
-      install_requires=['future', 'configparser;python_version<"3.2"',
+      install_requires=['configparser;python_version<"3.2"',
```

Comment:
This can be removed as Python < 3.2 is not supported here anymore.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/0984273e60b52bf8e6d1afcada2b175f5afbb4d8